mailpoet/assets/js/src/public.jsx

```diff
@@ -299,88 +388,36 @@ jQuery(($) => {
           }, 2500);
           return false;
         }
-        const formData = form.mailpoetSerializeObject() || {};
+
         // check if we're on the same domain
         if (isSameDomain(window.MailPoetForm.ajax_url) === false) {
           // non ajax post request
           return true;
         }
 
+        const formData = form.mailpoetSerializeObject() || {};
+        const size = form.children('.mailpoet_recaptcha').attr('data-size');
+
```

Comment:
Could you reconfirm you are getting the right value for the `data-size` attribute here?

When using the Invisible ReCaptcha option, I was getting `size` as undefined when it should be 'invisible'

Reply:
See https://github.com/mailpoet/mailpoet/pull/4084#issuecomment-1110270323